### PR TITLE
upgrade(ipc): Update node-ipc 8.9.2 -> 9.1.1

### PR DESCRIPTION
**Pull request by jhermsmeier**
Todo:

- [ ] Check if `disconnect` events work with reconnects now

Change-Type: patch

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -4682,7 +4682,7 @@
         "form-data": {
           "version": "0.2.0",
           "from": "form-data@>=0.2.0 <0.3.0",
-          "resolved": "http://registry.npmjs.org/form-data/-/form-data-0.2.0.tgz",
+          "resolved": "https://registry.npmjs.org/form-data/-/form-data-0.2.0.tgz",
```
Comment (lurch) [inline npm-shrinkwrap.json]: Weird how a bunch of these have spontaneously switched to `https:` ?

Comment (lurch) [reply npm-shrinkwrap.json]: ...and that change is now causing errors on Travis :sob:

Comment (jhermsmeier) [reply npm-shrinkwrap.json]: Yeah, but they should've never been `http:` in the first place as far as I can see.

Comment (jviotti) [reply npm-shrinkwrap.json]: Lets omit the `https` -> `http` changes from this PR to make Travis happy and get this merged ASAP

**Comment by jviotti** [comment]
We should have probably included a changelog entry on this PR, like "Fix stuck in Starting... if there was an error when initializing the IPC server." I guess we can manually add it to the CHANGELOG.

**Pull request merged**